src/umpire/tpl/simpool/CMakeLists.txt

```diff
@@ -3,15 +3,21 @@ set (simpool_headers
   FixedSizePool.hpp
   StdAllocator.hpp)
 
+set (simpool_sources
+  DynamicSizePool.cpp)
+
 blt_add_library(
   NAME umpire_tpl_simpool
-  HEADERS ${simpool_headers})
+  SOURCES ${simpool_sources}
+  HEADERS ${simpool_headers}
+  DEPENDS_ON umpire_util
```

Comment:
Merge `DynamicSizePool` into `umpire::strategy::DynamicPool` and remove simpool altogether.